README.md

```diff
@@ -18,6 +18,15 @@
 
 <hr />
 
+<div align="center">
+
+💡**Rasa Open Source 3.0 is coming up!**<br>
```

Comment:
```suggestion
💡**We are starting to work on Rasa Open Source 3.0!**<br>
```

Reply:
I want to make sure we don't signal that 3.0 might be here sooner than it actually will be

Reply:
sounds good